sql_generators/baseline_clients_city_seen_v1/__init__.py

```diff
@@ -0,0 +1,151 @@
+"""Generate baseline clients city seen per app."""
+
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, List
+
+import click
+from jinja2 import Environment, FileSystemLoader
+
+from bigquery_etl.cli.utils import use_cloud_function_option
+from bigquery_etl.config import ConfigLoader
+from bigquery_etl.format_sql.formatter import reformat
+from bigquery_etl.util.common import render, write_sql
+from sql_generators.glean_usage import get_app_info
+
+THIS_PATH = Path(__file__).parent
+TABLE_NAME = THIS_PATH.name
+BASE_NAME = "_".join(TABLE_NAME.split("_")[:-1])
+
+
+def get_app_info_map() -> Dict[str, List[str]]:
+    """Return a dict where key is the app name and the value a list of all the channel app id(s).
+
+    e.g. {
+        "firefox_desktop": ["firefox_desktop"],
+        "fenix": [
+            "org_mozilla_firefox",
+            "org_mozilla_fenix_nightly",
+            'org_mozilla_firefox_beta',
+            'org_mozilla_fenix',
+            'org_mozilla_fennec_aurora'
+        ],
+    }
+    """
+    app_info = get_app_info()
+    app_info = [
+        info
+        for name, info in app_info.items()
+        if name
+        in ConfigLoader.get(
+            "generate", "baseline_clients_city_seen", "apps", fallback=[]
+        )
```

Comment:
Assuming the allowlist is just for this initial phase, when you add the rest of the apps, you'll also need to check if the app has baseline pings since glean.js and server glean doesn't use them. That doesn't need to be done now but might be needed later.

One way to do that would be (I didn't test this):
```py
"glean-core" in requests.get(
  f"https://probeinfo.telemetry.mozilla.org/glean/{info[0]['v1_name']}/dependencies"
).json()
```
glean-usage downloads the schemas but that's slower if you only need to check baseline.